nautobot_chatops_ipfabric/ipfabric.py

```diff
@@ -324,3 +324,46 @@ def get_wireless_ssids(self, snapshot_id="$last", limit=DEFAULT_PAGE_LIMIT):
         }
 
         return self.get_response("/api/v1/tables/wireless/radio", payload)
+
+    def get_host(self, search_key, search_value, snapshot_id="$last", limit=DEFAULT_PAGE_LIMIT):
+        """Return inventory host information."""
+        logger.debug("Received host inventory request")
+
+        # columns and snapshot required
+        payload = {
+            "columns": [
+                "ip",
+                "vrf",
+                "dnsName",
+                "siteName",
+                "edges",
+                "gateways",
+                "accessPoints",
+                "mac",
+                "vendor",
+                "vlan",
+            ],
+            "filters": {search_key: ["eq", search_value]},
+            "pagination": {"limit": limit, "start": 0},
+            "snapshot": snapshot_id,
+        }
+        logger.debug("Requesting host inventory with payload: %s", payload)
+        return self.get_response("/api/v1/tables/addressing/hosts", payload)
+
+    def find_host(self, search_key, search_value, snapshot_id="$last", limit=DEFAULT_PAGE_LIMIT):
+        """Get and parse inventory host information."""
+        logger.debug("Parsing host inventory request")
+
+        hosts = self.get_host(search_key, search_value, snapshot_id, limit)
+        logger.debug("Parsing hosts: %s", hosts)
+        parsed_hosts = []
+
+        for host in hosts:
+            for edge in host.get("edges"):
+                edge_details = " ".join([edge.get("hostname", ""), edge.get("intName", "")])
+                host["edges"] = edge_details
```

Comment:
I'm a bit confused by these `fors`. It looks like we can get multiple edges and gateways, but we only store the last?? 🤔 
I guess I'm missing something here

Reply:
For all the data I have only one object is ever returned for these attributes. Will update to account for possible other elements.